src/screens/Auth/LoginStack/ConfirmResetCodeScreen/ConfirmResetCodeScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import {
+  Text,
+  View,
+  TouchableOpacity,
+} from 'react-native'
+import styles from './styles'
+import BlackHeading from '../../../../shared_components/BlackHeading/BlackHeading'
+import GreyTextNumInput from '../../../../shared_components/GreyTextNumInput/GreyTextNumInput'
+import PinkButton from '../../../../shared_components/PinkButton/PinkButton'
+import { NavigationScreenProp, NavigationState } from 'react-navigation'
+import { BASE_PATH } from 'react-native-dotenv'
+
+interface Props {
+  navigation: NavigationScreenProp<NavigationState>,
+}
+
+export default function ConfirmResetCodeScreen(props: Props) {
+  const [first, setFirst] = useState('')
+  const [second, setSecond] = useState('')
+  const [third, setThird] = useState('')
+  const [fourth, setFourth] = useState('')
+  const [message, setMessage] = useState('')
+  const [messageStyle, setMessageStyle] = useState(styles.messageError)
+  const {navigate} = props.navigation 
+  const user = {
+    email: props.navigation .state.params.email,
+  }
+
+  async function checkVerificationCode(){
+    fetch(`${BASE_PATH}/api/auth/confirm-reset-code`, {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        'Email': user.email,
+        'Code': first + second + third + fourth,
+      }),
+    })
+      .then(res => res.json())
+      .then(data => {
+        console.log(data)
+        if(data.statusCode == 401) {
+          setMessage('Error: Verification Code is invalid.')
+          setMessageStyle(styles.messageError)
+        } else {
+          navigate('CreateResetPassword', {
+            email: user.email,
+          })
```

Comment:
make this another if statement (if statusCode == 200), then the else displays something like 
`Error: Something went wrong - ${data.message}`